components/brave_rewards/common/mojom/bat_ledger.mojom

```diff
@@ -257,22 +257,7 @@ interface LedgerClient {
   ClearState(string name) => ();
 
   [Sync]
-  GetBooleanOption(string name) => (bool value);
-
-  [Sync]
-  GetIntegerOption(string name) => (int32 value);
-
-  [Sync]
-  GetDoubleOption(string name) => (double value);
-
-  [Sync]
-  GetStringOption(string name) => (string value);
-
-  [Sync]
-  GetInt64Option(string name) => (int64 value);
-
-  [Sync]
-  GetUint64Option(string name) => (uint64 value);
+  GetIsBitFlyerRegion() => (bool value);
```

Comment:
Would just `IsBitFlyerRegion` work?

Reply:
Yeah, that's intentional, as we already have an `IsBitFlyerRegion()` function in `RewardsServiceImpl`, which would hide the Mojom-generated virtual function:
```
In file included from ../../brave/browser/brave_rewards/rewards_service_factory.cc:16:
../..\brave/components/brave_rewards/browser/rewards_service_impl.h(601,8): error: 'brave_rewards::RewardsServiceImpl::IsBitFlyerRegion' hides overloaded virtual function [-Werror,-Woverloaded-virtual]
  bool IsBitFlyerRegion() const;
       ^
gen\brave/components/brave_rewards/common/mojom/bat_ledger.mojom.h(1169,16): note: hidden overloaded virtual function 'ledger::mojom::LedgerClient::IsBitFlyerRegion' declared here: different number of parameters (1 vs 0)
  virtual bool IsBitFlyerRegion(bool* out_value);
               ^
1 error generated.
```

Reply:
I would merge these two functions, as they're trying to achieve the same purpose. It's just that `GetExternalWalletType()` does not really do what it advertises, as it might override what's coming from the pref (`brave.rewards.external_wallet_type`) if `IsBitFlyerRegion() == true`. `GetIsBitFlyerRegion()` should inspect the type of the wallet directly, but that might have unwanted consequences elsewhere, which I wouldn't want to address in this PR.

Reply:
Oh, gotcha. I think `GetIsBitflyerRegion` and `IsBitflyerRegion` are very similar as-is. Not sure what the right strategy is, but we might need some time to think it through.

Reply:
One simple option (not ideal) would be to rename the current one - maybe just `IsBitflyerCountry` would be enough?

Reply:
Sure.